x-pack/plugin/esql/qa/testFixtures/src/main/resources/eval.csv-spec

```diff
@@ -366,12 +366,31 @@ FROM employees
 
 // tag::length-result[]
 first_name:keyword   | last_name:keyword   | fn_length:integer
-Alejandro      |McAlpine       |9              
-Amabile        |Gomatam        |7              
+Alejandro      |McAlpine       |9
+Amabile        |Gomatam        |7
 Anneke         |Preusig        |6
 // end::length-result[]
 ;
 
+docsByteLength
+required_capability: fn_byte_length
+// tag::byteLength[]
+FROM employees
+| KEEP first_name, last_name
+| EVAL fn_byte_length = BYTE_LENGTH(first_name)
+// end::byteLength[]
+| SORT first_name
+| LIMIT 3
+;
+
+// tag::byteLength-result[]
+first_name:keyword   | last_name:keyword   | fn_byte_length:integer
+Alejandro      |McAlpine       |9
```

Comment:
Would be nice to use an example where byte_length and length give different results.

Reply:
I used `airports` since that one has unicode text in it. I think it makes sense to have a single query with 4 columns (original text, length, bit length and byte length) instead of it being split into multiple examples. WDYT?